collect_app/src/androidTest/java/org/odk/collect/android/support/rules/CollectTestRule.kt

```diff
@@ -0,0 +1,69 @@
+package org.odk.collect.android.support.rules
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import org.odk.collect.android.activities.SplashScreenActivity
+import org.odk.collect.android.external.AndroidShortcutsActivity
+import org.odk.collect.android.support.pages.FirstLaunchPage
+import org.odk.collect.android.support.pages.MainMenuPage
+import org.odk.collect.android.support.pages.Page
+import org.odk.collect.android.support.pages.ShortcutsPage
+import java.util.function.Consumer
+
+class CollectTestRule @JvmOverloads constructor(
+    private val useDemoProject: Boolean = true
+) : TestRule {
+
+    override fun apply(base: Statement, description: Description): Statement {
+        return object : Statement() {
+            @Throws(Throwable::class)
```

Comment:
Let's get rid of it if it throws something it throws it and we do not catch it anywhere so I don't like those annotations in rules.

Reply:
Done.